.ast-grep/multicast-rule-require-docstring-testing.yml

```
@@ -0,0 +1,22 @@
+# .ast-grep/multicast-rule-require-docstring-testing.yml
+id: require-docstring-testing
+rule:
+  kind: class_definition
+  inside: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)